gaphor/ui/greeter.py

```diff
@@ -8,25 +8,55 @@
 from gaphor.action import action
 from gaphor.babel import translate_model
 from gaphor.core import event_handler
-from gaphor.diagram.hoversupport import flowbox_add_hover_support
 from gaphor.event import ActiveSessionChanged, SessionCreated
 from gaphor.i18n import gettext, translated_ui_string
 from gaphor.ui import APPLICATION_ID
 
 
 class ModelTemplate(NamedTuple):
     name: str
+    description: str
     icon: str
     lang: str
     filename: str
 
 
 TEMPLATES = [
-    ModelTemplate(gettext("Generic"), "org.gaphor.Gaphor", "UML", "blank.gaphor"),
-    ModelTemplate(gettext("UML"), "UML", "UML", "uml.gaphor"),
-    ModelTemplate(gettext("SysML"), "SysML", "SysML", "sysml.gaphor"),
-    ModelTemplate(gettext("RAAML"), "RAAML", "RAAML", "raaml.gaphor"),
-    ModelTemplate(gettext("C4 Model"), "C4Model", "C4Model", "c4model.gaphor"),
+    ModelTemplate(
+        gettext("Generic"),
+        gettext("An empty model."),
+        "org.gaphor.Gaphor",
+        "UML",
+        "blank.gaphor",
+    ),
+    ModelTemplate(
+        gettext("UML"),
+        gettext("A Unified Modeling Language template/example."),
+        "UML",
+        "UML",
+        "uml.gaphor",
+    ),
+    ModelTemplate(
+        gettext("SysML"),
+        gettext("Systems Modeling template"),
```

Comment:
I think we should either spell them all out completely or abbreviate them all. SysML is Systems Modeling Language and RAAML is Risk Analysis and Assessment Modeling Language. Why does the UML one say template/example and the others just say template?

Reply:
I quickly dropped some text.

RAAML spelled out is a bit too much text IMHO. Currently the description does not add a lot. 

Something like "Systems Modeling Language template" does not say anything about what you can do with it. I think that the term "template" is also a bit misplaced. You're creating a new model _based on a template_, not the template.